lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -13,6 +13,9 @@
 end
 
 require "zlib"
+require "open3"
+
+require_relative "lockfile_diff_parser"
```

Comment:
This _should_ be the same as:
```suggestion
require "tapioca/lockfile_diff_parser"
```
(for consistency with how we use `require` elsewhere in the codebase)